mk/spksrc.copy.mk

```diff
@@ -45,18 +45,17 @@ copy_target: SHELL:=/bin/bash
 copy_target: $(PRE_COPY_TARGET) $(INSTALL_PLIST)
 ifeq ($(call version_ge, ${TCVERSION}, 7.0),1)
 	@$(MSG) [DSM7+] Copy target to staging, discard var directory
-	@(mkdir -p $(STAGING_DIR) && cd $(INSTALL_DIR)/$(INSTALL_PREFIX) && tar cpf - `cat $(INSTALL_PLIST) | sed -e '/^.*:var\/.*/d' -e 's/^.*://g'`) | \
+	@(mkdir -p $(STAGING_DIR) && cd $(INSTALL_DIR)/$(INSTALL_PREFIX) && tar cpf - $$(cat $(INSTALL_PLIST) | sed -e '/^.*:var\/.*/d' -e 's/^.*://g')) | \
 	  tar xpf - -C $(STAGING_DIR)
-	@$(MSG) "[DSM7+] Copy & merge var and target/var to $(STAGING_DIR)/var"
-	@if [ "`cat $(INSTALL_PLIST) | sed -n 's?^.*:var/??p'`" ] ; then \
+	@$(MSG) "[DSM7+] Copy and merge var and target/var to $(STAGING_DIR)/var"
+	@if [ "$$(cat $(INSTALL_PLIST) | sed -n 's?^.*:var/??p')" ] ; then \
 	  mkdir -p $(STAGING_DIR)/var ; \
-	  (cd $(INSTALL_DIR)/$(INSTALL_PREFIX) && tar cpf - $$(eval ls -d $$(cat $(INSTALL_PLIST) | sed -n 's?^.*:var/??p' | sed -e 's?^?{var,target/var}/?') 2>/dev/null)) | \
-	    tar xpf - -C $(STAGING_DIR)/var --strip-components=1 --transform='s!^target/!!' ; \
+	  ([ -d $(INSTALL_DIR)/$(INSTALL_PREFIX)/../var ] && (cd $(INSTALL_DIR)/$(INSTALL_PREFIX)/../var && tar cpf - $$(eval ls -d $$(cat $(INSTALL_PLIST) | sed -n 's?^.*:var/??p') 2>/dev/null)) | tar xpf - -C $(STAGING_DIR)/var) || true ; \
+	  ([ -d $(INSTALL_DIR)/$(INSTALL_PREFIX)/var ] && (cd $(INSTALL_DIR)/$(INSTALL_PREFIX)/var && tar cpf - $$(eval ls -d $$(cat $(INSTALL_PLIST) | sed -n 's?^.*:var/??p') 2>/dev/null)) | tar xpf - -C $(STAGING_DIR)/var) || true ; \
```

Comment:
BTW, I'll give it a shot using your PR tomorrow but I wasn't able to replicate your issue.

Reply:
I was able to reproduce your issue.  Reason is fairly simple: path is wrong, missing a `/../` suffix (how did I missed that?!).  Fix pending here #5214 (currently testing against other var owned packages).

Your proposed fix is insuficient e.g. it will not confirm if you actually did hit an error related to `PLIST` file.  I've kept all your other smaller changes related converting backticks to `$$(..)`.

Reply:
Testing completed now merged.